oauth2/handler.go

```diff
@@ -59,6 +61,9 @@ const (
 	IntrospectPath   = "/oauth2/introspect"
 	RevocationPath   = "/oauth2/revoke"
 	DeleteTokensPath = "/oauth2/tokens" // #nosec G101
+
+	// Device Grant Handler
+	DeviceAuthPath = "/oauth2/device/auth"
```

Comment:
Additionally, I think it might be useful to handle user POST against the `verification_url` (https://datatracker.ietf.org/doc/html/rfc8628#section-3.3) so that a UI can render a simple form that will submit the `user_code` against the same endpoint.

Additionally, I think it might be useful to be able to configure a shorter version of this URL. `/oauth2/device/auth` is already quite long if you have to type it on a mobile, plus some origin. How about `/device` (as in the spec), or just `/dev`, or `/oauth2/dev` if you want to stick with the namespacing? But really, it should be configurable IMHO.

WDYT?

Reply:
> Additionally, I think it might be useful to handle user POST against the verification_url (https://datatracker.ietf.org/doc/html/rfc8628#section-3.3) so that a UI can render a simple form that will submit the user_code against the same endpoint.

Please ignore this remark, I now understand your approach with the device verifier, which is using a POST to the UI and then an admin API to bind the device to the flow.